### ISSUE TITLE
will open the big model config of ViT-Large-900q ?

### ISSUE THREAD

**Issue by FDInSky** [exiawsh/StreamPETR#27]

**Comment by exiawsh** [comment]
Sorry, the config of ViT-Large-900q is still kept private.

**Comment by FDInSky** [comment]
how many epochs did you train?

**Comment by exiawsh** [comment]
> how many epochs did you train?
less than 48.

**Comment by FDInSky** [comment]
Could you open the model(based on ViT-Large-900q) eval result ?

**Comment by exiawsh** [comment]
@FDInSky Sorry, we can't provide more details.

**Comment by wzc9401** [comment]
> @FDInSky Sorry, we can't provide more details.

Hi, can you share the results of ViT-Large on nuscenes val ?

**Comment by exiawsh** [comment]
> > @FDInSky Sorry, we can't provide more details.
> 
> @exiawsh Hi, can you share the results of ViT-Large on nuscenes val ?

@wzc9401 
The image resolution is 320x800
# Evaluating bboxes of pts_bbox
# mAP: 0.5361                                                                                                                                                                                                                  
# mATE: 0.5371
# mASE: 0.2531
# mAOE: 0.2780
# mAVE: 0.2364
# mAAE: 0.1987
# NDS: 0.6177
# Eval time: 139.1s

# Per-class results:
# Object Class    AP      ATE     ASE     AOE     AVE     AAE
# car     0.701   0.364   0.144   0.052   0.217   0.192
# truck   0.487   0.576   0.181   0.061   0.181   0.190
# bus     0.573   0.648   0.185   0.051   0.457   0.306
# trailer 0.301   0.912   0.224   0.421   0.171   0.145
# construction_vehicle    0.234   0.843   0.426   0.794   0.141   0.333
# pedestrian      0.594   0.527   0.284   0.334   0.253   0.139
# motorcycle      0.552   0.508   0.245   0.258   0.342   0.278
# bicycle 0.555   0.398   0.255   0.403   0.129   0.008
# traffic_cone    0.701   0.290   0.308   nan     nan     nan
# barrier 0.663   0.306   0.278   0.129   nan     nan

**Comment by wzc9401** [comment]
@exiawsh thanks, but I find the best results on test set is NDS 67.6 mAP 62, how about the result of the same model in the val set?

**Comment by exiawsh** [comment]
> @exiawsh thanks, but I find the best result on test set is NDS 67.6 mAP 62, how about the result of the same model in the val set?

We did not do additional ablation study with large image input on the validation set, and based on my own experience, it should be around 57mAP at high resolution input.

**Comment by exiawsh** [comment]
@zen-d 24epoch
